src/main/webapp/courseLoader.js

```diff
@@ -1,127 +1,200 @@
+let termData;
```

Comment:
This is a global which is a bit scary. Are you sure you need this to be global?

If anyone redeclares termData anywhere in the page's other JavaScript (or the user redefines window.termData) this will get changed.

Reply:
If you need it for this whole file, consider using an IIFE https://flaviocopes.com/javascript-iife/ using the arrow function variety means the rest of the code shouldn't need to be changed, just wrapped inside of it.

Reply:
Okay this is helpful because in a later PR I chose to just store variables like schoolName, termName, etc as global because they don't change and I uses them everywhere.